### PR TITLE
add some natives

### PR DESCRIPTION
**Pull request by MRV6**

### CODE REVIEW

**Comment by alloc8or** [comment]
What's with that random merge commit?

File: natives.json [hunk 1]
```diff
@@ -37774,15 +37774,15 @@
 			"gta_jhash": "0xAE92DD96"
 		},
 		"0x9FA00E2FC134A9D0": {
-			"name": "_0x9FA00E2FC134A9D0",
+			"name": "_DOES_ENTITY_BLIP_EXIST",
```
Comment (alloc8or) [inline natives.json:37777]: ``_DOES_ENTITY_HAVE_BLIP`` probably more fitting

File: natives.json [hunk 2]
```diff
@@ -37812,15 +37812,15 @@
 			"build": "1207"
 		},
 		"0xE9F676788F8D5E1E": {
-			"name": "_0xE9F676788F8D5E1E",
+			"name": "_IS_BLIP_ON_MAP",
```
Comment (alloc8or) [inline natives.json:37815]: This does not check whether the blip is on the map. It actually checks if the blip is attached to any entity.
So ``_IS_BLIP_ATTACHED_TO_ANY_ENTITY`` would be better.

File: natives.json [hunk 3]
```diff
@@ -85781,27 +85781,27 @@
...
 			"params": [
 				{
-					"type": "Any",
-					"name": "p0"
+					"type": "Vehicle",
+					"name": "vehicle"
 				}
 			],
-			"return_type": "Any",
+			"return_type": "int",
```
Comment (alloc8or) [inline natives.json:85792]: Please change this to ``PropSet``.

**Review by alloc8or — CHANGES_REQUESTED** [review]

**Comment by MRV6** [comment]
> What's with that random merge commit?
I did this to avoid conflict thinking it was necessary

**Comment by MRV6** [comment]
Anyway, changes done

**Pull request merged**